e2e/gethdevnet.go

```diff
@@ -33,7 +34,8 @@ func gethdevnet(blockTime uint64, genesis *core.Genesis) (*rpc.Client, string, e
 	if err != nil {
 		return nil, "", fmt.Errorf("start geth L1: %w", err)
 	}
-	// TODO close node
+
+	env.DeferErr(fmt.Errorf("close geth L1: %w", err).Error(), node.Close)
```

Comment:
We don't want to wrap a `nil` error.
```suggestion
	env.DeferErr("close geth node", node.Close)
```